modules/client/src/allowlistVoting/client.ts

```diff
@@ -0,0 +1,49 @@
+import {
```

Comment:
Why renaming the entire folder and losing track of all history? It's impossible to spot what changed

Reply:
because I did a complete rewrite of the code. also, the name of the folder was wrong. The plugin is called AllowlistVoting and not addresslist